bot.ts

```diff
@@ -123,26 +123,31 @@ async function makeReplacements(
   text: string,
 ): Promise<[string, MessageEntity[], string]> {
   const searchQueries = new Array<string>();
+  text = `${ZWSP}${text}`;
   text.replace(replacementExp, (_, s, o) => {
     s = s.split("|");
     searchQueries.push(s[0]);
     return s;
   });
-  const urls = new Array<string>();
+  const hits = new Array<any>();
```

Comment:
Do you want to reuse the type definitions I added, rather than removing more and more?